### DISCUSSION TITLE
Only alive patients

### DISCUSSION THREAD

**Discussion by chintudwh** [synthetichealth/synthea#1264] [Q&A] [answered]
Hi,

what is the command to generate only alive patients? I am trying to generate 1000 patients and export to CSV. Is the below command correct? I appreciate your help.

java -jar synthea-with-dependencies.jar -p 1000 --onlyAlivePatients true --exporter.csv.export true

Thanks

**Comment by jawalonoski** [comment, marked as answer]
Use `--generate.only_alive_patients=true`

Browse through this file: https://github.com/synthetichealth/synthea/blob/master/src/main/resources/synthea.properties

**Reply by chintudwh** [reply]
Awesome! thank you so much.
